src/shapes/textbox.class.ts

```diff
@@ -44,6 +44,14 @@ export class Textbox extends IText {
    */
   splitByGrapheme: boolean;
 
+  /**
+   * Implementation of css property overflow-wrap.
+   * unlike splitByGrapheme, words will have reasonable newlines
+   * Will block splitByGrapheme option!
+   * @type Boolean
+   */
+  overflowBreakWord: boolean;
```

Comment:
IMO we should introduce a `breakLineStrategy` prop or something similar instead of a flag and perhaps make `splitByGrapheme` an option

Reply:
I agree, but how to deal with the previous versions of `splitByGrapheme` ?

Reply:
We can mark it as `deprecated` and introduce its predecessor and have logic supporting falling back to `splitByGrapheme`.
Or we break.
@asturur ?

Reply:
introduced `resolveTextOverflowStrategy`